In musl libc _XOPEN_SOURCE is already defined in <features.h>

--- src/eog-util.c	2015-09-28 14:46:13.000000000 +0200
+++ src/eog-util.c	2015-11-06 16:15:18.389300086 +0100
@@ -28,7 +28,9 @@
 
 #include <sys/time.h>
 #ifdef HAVE_STRPTIME
+#ifndef _XOPEN_SOURCE
 #define _XOPEN_SOURCE
+#endif
 #endif /* HAVE_STRPTIME */
 
 #include <time.h>
